coldfront/plugins/qumulo/management/commands/test_connections.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import ldap3
+from qumulo.rest_client import RestClient
+
+import os
+from dotenv import load_dotenv
```

Comment:
@harterjwustl  did you mean to invoke `load_dotenv`?

Reply:
good catch, I don't think I need this, no